Docs/performance-tests.md

```diff
@@ -0,0 +1,46 @@
+Performance tests
+==================
+
+There are several scenarios in which having performance tests to meassure how the app performs is really important. The data bases for Bookmarks, History, Tabs could be very big and impact in the time it takes to launch the app or use it. 
+
+Being able to pre-load existing data bases with heavy profiles has helped to implement these tests. It is possible to launch a test with a specific profile and meassure differnt data that could help to find issues for the app in that situation.
+
```

Comment:
s/data bases/databases

Reply:
sorry not sure where should I change/add that part... in Line 6 it says data bases as the second option in your comment...